core/vm/ovm_state_manager_test.go

```diff
@@ -57,9 +56,11 @@ func TestEthCallNoop(t *testing.T) {
 	if len(diff) > 0 {
 		t.Fatalf("map must be empty since it was an eth call")
 	}
+	os.Remove("test1")
```

Comment:
Could also `defer os.Remove("test1")` right underneath `makeEnv` to make it more go-like but not that big of a deal